.github/workflows/ci_windows.yml

```
@@ -7,16 +7,16 @@ on:
       - ".vscode/**"
       - "doc/**"
       - "*.md"
-  pull_request:
-    paths-ignore:
-      - ".devcontainer/**"
-      - ".vscode/**"
-      - "doc/**"
-      - "*.md"
+    branches:
+      - "dev"
+  pull_request_review:
+    types: [submitted, edited]
+  workflow_dispatch:
 
 jobs:
   gcc8-build:
     name: GCC8 build
+    if: github.event.review.state == "approved"
```

[公式ドキュメント](https://docs.github.com/ja/actions/using-workflows/events-that-trigger-workflows) にはシングルクォーテーションで区切る形にされているので、それに一旦合わせるとうまく行くかもしれないです。
```suggestion
    if: github.event.review.state == 'approved'
```